pahfit/base.py

```diff
@@ -11,6 +11,8 @@
 
 import matplotlib as mpl
 
+from pahfit.feature_strengths import feature_strength, line_strength
```

Comment:
very similar names feature_strengths and feature_strength

Reply:
Changed the function to pah_feature_strengths.